packages/manager/modules/pci/src/projects/project/failover-ips/imports/import/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+import uiRouter from '@uirouter/angularjs';
+
+import routing from './import.routing';
+
+const moduleName = 'ovhManagerPciProjectFailoverIpImport';
```

Comment:
Shouldn't be `ovhManagerPciProjectFailoverIpsImportsImport` instead?

```suggestion
const moduleName = 'ovhManagerPciProjectFailoverIpsImportsImport';
```